client-sdk/ts-web/rt/src/consensus_accounts.ts

```diff
@@ -0,0 +1,66 @@
+import * as oasis from '@oasisprotocol/client';
+
+import * as types from './types';
+import * as wrapper from './wrapper';
+
+/**
+ * Unique module name.
+ */
+export const MODULE_NAME = 'consensus_accounts';
+
+export const ERR_INVALID_ARGUMENT_CODE = 1;
+export const ERR_INVALID_DENOMINATION_CODE = 2;
+export const ERR_INSUFFICIENT_WITHDRAW_BALANCE_CODE = 3;
+
+// Callable methods.
+export const METHOD_DEPOSIT = 'consensus.Deposit';
+export const METHOD_WITHDRAW = 'consensus.Withdraw';
+// Queries.
+export const METHOD_BALANCE = 'consensus.Balance';
+export const METHOD_ACCOUNT = 'consensus.Account';
+
+/**
+ * Consensus deposit call.
+ */
+export interface ConsensusDeposit {
+    amount: types.BaseUnits;
+}
+
+/**
+ * Consensus withdraw call.
+ */
+export interface ConsensusWithdraw {
+    amount: types.BaseUnits;
+}
+
+/**
+ * Consensus balance query.
+ */
+export interface ConsensusBalanceQuery {
+    address: Uint8Array;
+}
+
+/**
+ * Consensus account balance.
+ */
+export interface AccountBalance {
+    balance: Uint8Array;
+}
```

Comment:
:two: I had been putting these data types in `types.ts`. Is that okay with you?

Reply:
Oh yeah sure i can move these